tests/trustedtypes_test.js

```diff
@@ -562,6 +552,14 @@ describe('v2 TrustedTypes', () => {
       expect(TrustedTypes.getPolicyNames()).not.toContain('default');
       expect(TrustedTypes.getPolicyNames()).toContain('a');
     });
+
+    it('allows policy collision only when allowed policies includes *', () => {
```

Comment:
Can you create 2 separate test cases? One with `setAllowedPolicyNames(['*'])` and another one with `setAllowedPolicyNames(['foo'])` ?

Reply:
Yeah, good idea. I split the tests and also moved them in a separate `describe`.